code/modules/mob/living/living_defense.dm

```diff
@@ -40,8 +40,10 @@
 	var/impact_damage = (1 + O.throwforce*THROWFORCE_COEFF)*O.throwforce*THROW_SPEED_IMPACT_COEFF*O.cur_speed
 
 	var/datum/launch_metadata/LM = O.launch_metadata
+	var/launch_meta_valid = istype(LM)
```

Comment:
I also don't understand this bit - why use istype() for this? Can launch_metadata be another type of value? At a glance it looks to me like using ?. would be enough

Reply:
Replication of LM handling I saw elsewhere, so standardized across the board. If you prefer I can just `if(!LM)` everywhere? Examples on master: https://github.com/search?q=repo%3Acmss13-devs%2Fcmss13%20istype(LM)&type=code